exercises/practice/complex-numbers/.meta/config.json

```diff
@@ -14,7 +14,8 @@
     "SleeplessByte",
     "Smarticles101",
     "sshine",
-    "Wasy18"
+    "Wasy18",
+    "jagdishdrp"
```

Comment:
This list is alphabetically sorted. Could you please move your username entry to maintain the order?

Reply:
Sure! Thank you for your insights.

Reply:
To be honest, I haven't tested, _but_ this should actually be your Github handle (according to the [documentation](https://exercism.org/docs/building/tracks/practice-exercises)).

Reply:
I am sorry for not reading this part of the documentation prior to this, I will change this to my GitHub handle. Thank you again!

Reply:
This time, I've included my GitHub handle, and I believe this should work.